packages/swingset-liveslots/src/collectionManager.js

```diff
@@ -293,6 +293,17 @@ export function makeCollectionManager(
       return `${dbKeyPrefix}${dbEntryKey}`;
     }
 
+    // a "vref" is a string like "o-4"
+    // an "EncodedKey" is the output of encode-passable:
+    // * strings become `s${string}`, like "foo" -> "sfoo"
+    // * positive BigInts become `p${len}:${digits}`, 47n -> "p2:47"
```

Comment:
This isn't exactly right, as [documented in encodePassable.js](https://github.com/endojs/endo/blob/67141a177bef32e037f454a696a4532999560ff8/packages/marshal/src/encodePassable.js#L151-L162).
```suggestion
    // * small positive BigInts become `p${len}:${digits}`, like 47n -> "p2:47"
```